networking.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: page
+title: Making Networking Requests in Flutter
+permalink: /networking/
+---
+
+* TOC Placeholder
+{:toc}
+
+## Using the `http` package
+
+Flutter supports the the [http package][http], version `0.11.3+12` or later.
+
+Start by declaring a dependency by adding an http line under `dependencies` in
+`pubspec.yaml`:
+
+```
+dependencies:
+  ...
+  http: '>=0.11.3+12'
+```
+
+## Making requests
+
+You can create an HTTP [Client][client] with the `Client` constructor:
+
+<!-- skip -->
+```dart
+import 'package:http/http.dart' as http;
+
+http.Client _httpClient = new http.Client();
+```
+
+The client supports common HTTP operations:
+
+* **HTTP GET:** Use [`get`][get] for a general request, [`read`][read] for a
+ request returning a string, or [`readbytes`][readbytes] for a request
+ returning bytes.
+
+* **HTTP POST:** Use [`post`][post] for a general post.
+
+Note that the API uses [Dart
```

Comment:
Consider putting this after the code snippet, so it's more of a note.

As written, "the API" isn't clear what API you're talking about. But, after the code, you can refer to "the code above".